mod-audit-server/src/main/java/org/folio/dao/OrderEventDao.java

```diff
@@ -0,0 +1,12 @@
+package org.folio.dao;
```

Comment:
I propose to change package names org/folio/dao/OrderEventsDao -> org/folio/dao/acquisition/OrderEventsDao

Reply:
fixed